dtcli/scripts/dt.py

```diff
@@ -43,7 +44,26 @@
                             help="Ignore subtleties, overwrite without prompt, when in doubt - advance!")
 
 
-def validate_parse_subject(ctx, param, value):
+def validate_ca_subject(subjects: List[str]):
```

Comment:
Missing return type, should be dict of str into str (more specifically Enum of allowed certificate metadata fields into str)